gpii/node_modules/flowManager/src/ReadyGetHandler.js

```diff
@@ -41,6 +41,7 @@ gpii.flowManager.cloudBased.ready.handler.isSameHost = function (flowManagerHost
     // Remove the trailing slash if any
     prefsServerHost = prefsServerHost.replace(/\/$/, "");
     flowManagerHost = flowManagerHost.replace(/\/$/, "");
+    fluid.log("Preferences server: '", prefsServerHost, "', Flow manager: '", flowManagerHost, "'");
```

Comment:
Is this line for debugging? If so, please remove it. Thanks.

Reply:
Oops.  Yes, for debugging -- thanks for catching that.